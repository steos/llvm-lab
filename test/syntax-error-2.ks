
	void main() {
		if (true) foo();
	}