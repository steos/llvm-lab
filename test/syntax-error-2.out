Parse error in ./test/syntax-error-2.ks on line 3 at offset 12: missing token, expected BRACE_L
Parse error in ./test/syntax-error-2.ks on line 3 at offset 12: missing token, expected BRACE_R
